cmake/versions.json

```
@@ -1,8 +1,8 @@
 {
   "packages" : {
-    "libcudacxx" : {
-      "version" : "1.9.0",
-      "git_url" : "https://github.com/NVIDIA/libcudacxx.git",
+    "cccl" : {
+      "version" : "v2.2.0",
+      "git_url" : "https://github.com/NVIDIA/cccl.git",
```

```suggestion
    "cccl" : {
      "version" : "v2.2.0",
      "git_url" : "https://github.com/NVIDIA/cccl.git",
```
This needs to look like:
```
    "CCCL" : {
      "version" : "2.2.0",
      "git_url" : "https://github.com/NVIDIA/cccl.git",
      "git_tag" : "v${version}"
    }
```
As the version string should be numeric only, and the name is case sensitive ( needs to match the `find_package` name ).

---

It currently works due to your entry being ignored, and the default `CCCL` entry being used.